data/json/items/gunmod/underbarrel.json

```diff
@@ -237,6 +237,21 @@
     }
   },
   {
+    "id": "inter_bayonet",
+    "copy-from": "spike",
```

Comment:
This could stand to do more damage than the crappy `spike` item, as the spike is a pointy piece of scrap, whereas this bayonet is an actual manufactured weapon.